example/connection/main_waad.tf

```diff
@@ -0,0 +1,26 @@
+provider "auth0" {}
+
+resource "auth0_connection" "my_connection" {
```

Comment:
This is most likely to conflict with `main.tf`'s `my_connection` resource. Did you try the example? 

Also, maybe its better to just name the file `waad.tf` or even merge it with `main.tf` as it seems pretty common with other providers.